.travis.yml

```diff
@@ -1,76 +1,36 @@
 env:
   global:
-    - JAVA_HOME=/usr/lib/jvm/java-8-oracle
-    - APP_VERSION=$TRAVIS_TAG
     - PACKAGE_NAME=org.phidatalab.radar_armt
-    - PROJECT_NAME=RADAR-base/RADAR-Questionnaire
+    - KEYSTORE_PATH=radar-armt-release-key.keystore
+    - KEYSTORE_ALIAS=alias_name
+    - PLAYSTORE_SERVICE_KEY=service-api.json
+    - PLAYSTORE_TRACK=alpha
   matrix:
     secure: m8xkdQA8dhCgufxe1URYqeTuN55E5XzU22DmAJIpMvDhU3v7UfIYzTYCRJWBHxCHvi8rF3oh7ajfthVr763ZyJUSLuZ1kg1y1VTqAMedAdqr6LvGCOayBK7wxgHRPCNsIyHijxDChHB90mrPIbu6BkLMH4D94m+O1ishEoifMxwGEVp80/ftKfhkbXdyG8WBrgLLfGwTlK0xqprOAJGUelkYXCq7jZGzN3cf5kkAze+cEWa/2PAKdBlMmnLKuh6YEGzbRYNtQEr7QChCGizXVkoAuEv+OShSHwzhy7f4juSDQRyWDW92REJu+fO3vs+iuBG6BD7TzWb7HsqUzbmrnirnFYfMv/U/PYEnIaa9q91dqpqVaI2ZQ+1dHX66VppYaE4PbfMTXPls7qZfvGX94MtC2K5TQdJUCKNPZhWESJTS2UgwVfTAzMVpJvjd9vOfiiVcFo6qw4Lo04kv44M2p8Ivd7BIf0w2LsNayPnNb6kffcYUnchGruHiPPduKJXOTVE2R7MxlXjq6epIS7Onr4QNapmHf3HY/qWKTsLBSVkrngELOvQ9bwY3cChulXGMN7bXGrpPjkrYS8uaDb0k+qKftEM/MSKlLnsTAcdbPIj2aQF2L/mzP6d7MvaZJGziIIj4Ex6dnD2MhZXwXM4G43R8/7ZedlkKkF5OswtdBQE=
-dist: trusty
-sudo: required
 language: android
 android:
   components:
-    - tools
-    - platform-tools
-    - tools
     - build-tools-28.0.3
     - android-28
-    - extra-google-google_play_services
-    - extra-google-m2repository
-    - extra-android-m2repository
-addons:
-  apt:
-    sources:
-      - sourceline: deb https://dl.yarnpkg.com/debian/ stable main
-        key_url: https://dl.yarnpkg.com/debian/pubkey.gpg
-      - sourceline: deb http://dl.google.com/linux/chrome/deb/ stable main
-        key_url: https://dl-ssl.google.com/linux/linux_signing_key.pub
-    packages:
-      - yarn
-      - jq
-      - gradle
-      - google-chrome-stable
-licenses:
-  - android-sdk-preview-license-.+
-  - android-sdk-license-.+
-  - google-gdk-license-.+
-cache:
```

Comment:
why was caching removed? It would be good to cache yarn and bundler(https://docs.travis-ci.com/user/caching/#enabling-bundler-caching)

Reply:
I've added it back